ext/reflection/php_reflection.c

```diff
@@ -5335,7 +5335,7 @@ ZEND_METHOD(reflection_property, getValue)
 		zval rv;
 
 		if (!object) {
-			zend_type_error("No object provided for getValue() on instance property");
+			zend_argument_type_error(1, "must be provided in case of instance properties");
```

Comment:
```suggestion
			zend_argument_type_error(1, "must be provided for instance properties");
```

Maybe? (Current is also fine though.)

Reply:
It's better so I'm ok with the change!